create-aliquot.avsc

```diff
@@ -0,0 +1,84 @@
+{
+  "type": "record",
+  "name": "CreateAliquot",
+  "namespace": "uk.ac.sanger.psd",
+  "fields": [
+     {
+      "name": "messageUuid",
+      "doc": "Unique message ID.",
+      "type": "uuid"
+    },
+    {
+      "name": "messageCreateDateUtc",
+      "doc": "Date (UTC) that the message was created.",
+      "type": { 
+        "type": "long", 
+        "logicalType": "timestamp-millis" 
+      }
+    },
+    {
+      "name": "LimsId",
+      "type": "string",
+      "doc": "The LIMS system that the aliquot was created in"
+    },
+    {
+      "name": "LimsUuid",
+      "type": "uuid",
+      "doc": "The UUID of the aliquot in the LIMS system"
+    },
+    {
+      "name": "aliquotType",
+      "type": "enum",
+      "doc": "The type of the aliquot",
+      "symbols": ["primary", "derived"]
+    },
+    {
+      "name": "sourceType",
+      "type": "enum",
+      "doc": "The type of the source of the aliquot"
+      "symbols": ["well", "sample", "library", "pool"]
+    },
+    {
+      "name": "sourceBarcode",
+      "type": "string",
+      "doc": "The barcode of the source of the aliquot"
+    },
+    {
+      "name": "sampleName",
+      "type": "string",
+      "doc": "The name of the sample that the aliquot was created from"
+    },
+    {
+      "name": "usedByType",
+      "type": "enum",
+      "doc": "The type of the entity that the aliquot is used by"
+      "symbols": ["library", "pool", "run"]
+    },
+    {
+      "name": "usedByBarcode",
+      "type": "string",
+      "doc": "The barcode of the entity that the aliquot is used by"
+    },
+    {
+      "name": "volume",
+      "type": "float",
+      "doc": "The volume of the aliquot (uL)"
+    },
+    {
+      "name": "concentration",
+      "type": ["null", {
+        "type": "float"
+      }],
+      "doc": "The concentration of the aliquot (ng/ul)",
+      "default": null
+    },
+    {
+      "name": "recordedAt",
+      "type": {
+        "type": "long",
+        "logicalType": "timestamp-millis"
+      },
+      "doc": "The date and time that the aliquot was recorded"
+    }
+  ]
+}
```

Comment:
Not a big deal here since you're not merging this in, but you might want to look at your IDE's settings for adding a final new line character to files when saving.  Line 84 doesn't exist without one according to the standards for how a line in a text file is defined.